.gitignore

```diff
@@ -1 +1,3 @@
 /.sqitch
+
+.idea/
```

Comment:
This should probably be in your own, global gitignore file – it's not project-specific. 😇